symfony/framework-bundle/3.3/manifest.json

```diff
@@ -14,7 +14,10 @@
     "env": {
         "APP_ENV": "dev",
         "APP_DEBUG": "1",
-        "APP_SECRET": "%generate(secret)%"
+        "APP_SECRET": "%generate(secret)%",
+        "#1": "See https://symfony.com/doc/current/request/load_balancer_reverse_proxy.html",
+        "#TRUSTED_PROXIES": "0.0.0.0/0",
+        "#TRUSTED_HEADERS": "HEADER_FORWARDED"
```

Comment:
IMO, it should not be configurable using environment variables.

Anyway, it should default too `X_HEADER_FORWARDED`, almost all existing software (Varnish, Nginx) and Cloud Platform (Kubernetes) use this one for now.